README.md

```diff
@@ -23,7 +23,7 @@ into your serializer:
 ```javascript
 // app/serializers/post.js
 
-import { ActiveModelSerializer } from 'active-model-adapter';
+import { ActiveModelSerializer } from "active-model-adapter";
```

Comment:
I'd keep these single quote, since that's also the rule for the prettier config of this codebase and for new ember apps.

Reply:
good spot, for some reason prettier was changing them to double quotes and I hadn't noticed